core_chat_presentation/src/commonMain/kotlin/net/thechance/mena/core_chat/presentation/di/ApiModule.kt

```diff
@@ -0,0 +1,11 @@
+package net.thechance.mena.core_chat.presentation.di
+
+import net.thechance.mena.core_chat.api.CoreChatApi
+import net.thechance.mena.core_chat.presentation.api.CoreChatApiImp
+import org.koin.core.module.dsl.singleOf
+import org.koin.dsl.bind
+import org.koin.dsl.module
+
+internal val ApiModule = module {
```

Comment:
```suggestion
internal val apiModule = module {
```
why this only module starts with uppercase